docs/development/style-guide.md

```diff
@@ -21,6 +21,32 @@ Second sentence on a new line.
 And so on.
 ```
 
+## Avoid contractions/possessives
+
+Avoid:
+
+```markdown
+don't
+won't
+doesn't
+shouldn't
+wouldn't
+manager's
+file's
+```
+
+Do:
+
+```markdown
+do not
+will not
+does not
+should not
+would not
+```
+
+Avoid possessives like `manager's` or `file's`, if needed rewrite the sentence.
```

Comment:
Maybe we could add some examples? Most times manager's and bot's can be rewritten as `Renovate's manager` and `Renovate's` respectively.

Reply:
Hmm, I'm not sure about `manager's` being easy to rewrite. From a quick Renovate docs search for `manager's`, it looks like `manager's` mostly refers to any of Renovate's managers. Not to a specific "Renovate manager". I hope this makes sense.... Here are some examples of what I mean:

> Renovate searches in each repository for any files matching each manager's configured `fileMatch` pattern(s)

> #### Extending a manager's default fileMatch
> 
> If the default `fileMatch` regular expression for a manager does not match against one of your relevant files, you can _extend_ the existing regular expression(s) by configuring a manager's `fileMatch` like in this example:
> 
> ```json
> {
>   "dockerfile": {
>     "fileMatch": ["does-not-look-like-a-docker-file"]
>   }
> }
> ```

> But when such files need updating, Renovate does so by using the package managers themselves instead of trying to "reverse engineer" each package manager's file formats and behavior.